### DISCUSSION TITLE
[BUG] Can't put a post to series / 无法使用系列

### DISCUSSION THREAD

**Discussion by CXwudi** [HEIGE-PCloud/DoIt#228] [Q&A] [answered]
### Describe the bug 描述你遇到的错误

When I try to put `series: ["my-series"]` in the post markdown file, Hugo failed to build
把文章加进系列里导致hugo启动失败

### Expected behavior 期待的行为

Hugo should boot
Hugo应该可以build并启动带有系列的文章

### Screenshots 屏幕截图

![image](https://user-images.githubusercontent.com/17377423/133558522-aa76fd12-0012-4719-ac62-0869e1b8fb93.png)

### Build Environment 构建环境

- OS: WSL2 Ubuntu 20.04
- Theme version/commit: 5bf539e7ea056a994d06807c11bdce6bcf066499
- Hugo version: hugo v0.88.1+extended linux/amd64 BuildDate=unknown

### Preview Environment 预览环境

- OS: Windows 10
- Browser: Chrome

### Additional Information 补充信息

感觉是不是文档哪里没有补充系列具体怎么用

**Comment by HEIGE-PCloud** [comment, marked as answer]
在 `config.toml` 中添加分类设置
Add the following settings to your `config.toml`

```toml
[taxonomies]
    series = "series"
```

Reference: https://hugodoit.pages.dev/zh-cn/theme-documentation-basics/#site-configuration

**Reply by HEIGE-PCloud** [reply]
文档里确实没有 “如何添加新的分类” 这样的文章。目前文档仅能保证提供的示例配置文件是正确的。对于新用户来说，这并不会造成问题；但对于老用户来说，在升级版本后可能需要修改一些配置才可以使用新添加的功能，而目前的文档中并没有对于迁移与升级提供任何指南。在未来，这个问题可能会越来越严重。

**Reply by HEIGE-PCloud** [reply]
维护一份迁移指南的成本是非常高昂的，像“开启系列分类”这种功能又不值得单独写一篇 post。对于 Hugo 比较熟悉的用户可能能够自行[定位问题](https://gohugo.io/content-management/taxonomies/)。对于其他用户，我目前的打算还是继续利用 GitHub Discussions 进行回复。这样也可以帮助遇到同样问题的用户。

**Reply by CXwudi** [reply]
谢谢你的回答和讲解，也感谢你对doit主题所做的贡献，这主题也经历了几代人的接手，能接手过来就很不容易了。确实我是第一次使用hugo，而且一上来就想要好多好特性对于我这样的新手来说确实很难。可能我只是忘了更多地读一读官方文档了，昨天我只读了doit自带的文档。下次我先读一读相关的官方文档再提问。
